src/extensions/default/SVGCodeHints/SVGAttributes.json

```diff
@@ -11,6 +11,15 @@
     "animate/fill": {
         "attribOptions": ["freeze", "remove"]
     },
+    "animateColor/fill": {
+        "attribOptions": ["freeze", "remove"]
+    },
```

Comment:
I was saying that `animate*/fill` should only hint `freeze` and `remove`, so we can use `fill` for color names.

> I'm not sure what you mean.

```xml
"animate*/fill": {
    "attribOptions": ["freeze", "remove"]
},
"fill": {
    "type": "color"
},
```
Then we can use its type property for hinting color names.